README.md

```diff
@@ -34,89 +63,165 @@ message MyContext {
   // Add any fields you like here   
 }
 ```
-Next there are a few boilerplate items that you need to take care of on both the client and server side.
 
-### Client side setup:
+Next declare an instance of `KonigKontext` in a shared location. All of your microservices should have access to this
+value:
 
-The client side setup just has one step. Simply add the `KonigKontextClientInterceptor` to all of your gRPC clients:
+Kotlin:
 
 ```kotlin
-val myServiceClient = MyServiceCoroutineStub(myManagedChannel).withInterceptors(
-    KonigKontextClientInterceptor()
-)
+val MY_GLOBAL_KONTEXT: KonigKontext<MyContext> = KonigKontext.fromProtoMessage(MyContext::class)
```

Comment:
this feels a bit strange to me since you don't actually need this instance? you just need to use any instance as long as it's for the right type?

(I guess having it be an instance feels like there is some state in there, when there is not?)